routes/authenticate.js

```diff
@@ -0,0 +1,127 @@
+const { Router } = require("express");
+const router = Router();
+const User = require("../models/User.model");
+const bcrypt = require("bcryptjs");
+const app = require("../app");
+const salt = 10; // this is the number of times we are hashing the password
+
+//this is to stop someone who is already signed in from going to either the login or signup page, they are instantly redirected to the current page.
```

Comment:
In this case in particular, you redirect the user to the homepage (the "/" endpoint)